src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -1298,7 +1298,8 @@ static class B
             var comp = CreateCompilation(new[] { source, s_utils }, parseOptions: TestOptions.RegularPreview, options: TestOptions.ReleaseExe);
             if (expectedDiagnostics is null)
             {
-                CompileAndVerify(comp, expectedOutput: $"{expectedMethod}: {expectedType}");
+                // ILVerify: Unrecognized arguments for delegate .ctor.
```

Comment:
>ILVerify: Unrecognized arguments for delegate .ctor. [](http://example.com/codeflow?start=19&length=52)

What does this error refer to?

Reply:
Unfortunately, that's all the details from ILVerify at this point.